.gitignore

```diff
@@ -33,3 +33,4 @@ yarn-debug.log*
 /public/uploads
 /public/users
 /public/animations
+/app/assets/images
```

Comment:
which images you are going to add here?

Reply:
The images which i have used for html page